macros/utils/cross_db_utils/generate_elementary_profile_parameters.sql

```diff
@@ -0,0 +1,104 @@
+{% macro generate_elementary_profile_parameters(method=none, overwrite_values=none) %}
```

Comment:
Maybe rename to `generate_elementary_profile_args`?

Reply:
ok